sbin/common/downloaders.sh

```diff
@@ -49,7 +49,7 @@ function downloadLinuxBootJDK() {
   # the fallback mechanism, as downloading of the GA binary might fail.
   set +e
   curl -L -o bootjdk.tar.gz "${apiURL}"
-  apiSigURL=$(curl -v "${apiURL}" 2>&1 | tr -d \\r | awk '/^< Location:/{print $3 ".sig"}')
+  apiSigURL=$(curl -v "${apiURL}" 2>&1 | tr -d \\r | awk '/^< [Ll]ocation:/{print $3 ".sig"}')
```

Comment:
A case insensitive search would work too, but I'm ok with this.
```suggestion
apiSigURL=$(curl -v "${apiURL}" 2>&1 | tr -d \\r | awk '/^< Location:/i{print $3 ".sig"}')
```

Reply:
/i doesnt work on my fedora installation.. so I went with the regex fix.

Reply:
Little bit of script I wrote to test..

```
apiURL=https://api.adoptium.net/v3/binary/latest/22/ga/linux/riscv64/jdk/hotspot/normal/eclipse
##apiSigURL=$(curl -v "${apiURL}" 2>&1 | tr -d \\r | awk '/^< [Ll]ocation:/{print $3 ".sig"}')
apiSigURL=$(curl -v "${apiURL}" 2>&1 | tr -d \\r | awk '/^< Location:/i{print $3 ".sig"}')
echo 1 = $apiURL
echo 2 = $apiSigURL
curl -L -o bootjdk.tar.gz.sig "${apiSigURL}"
```